src/components/home-header.tsx

```diff
@@ -19,12 +19,15 @@ export const HomeHeaderTitle = () => {
       break
   }
 
-  const showLoading = loginState === LoginState.LoggingIn
   const {user} = useContext(UserContext)
 
+  const showLoading = loginState === LoginState.LoggingIn && user === undefined
```

Comment:
This seems like it should be an or for me?

Reply:
I agree it could be clearer. But it's correct as I don't want to  show loading animation if the header can access the cached offline user